charts/chronicle/templates/tests/define-agent.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Chart.Name }}-define-agent
+  annotations:
+    "helm.sh/hook": test
+spec:
+  template:
+    spec:
+      initContainers:
+        - name: gq
+          image: node:18
+          command: ["/bin/bash"]
+          args: ["-c", "scripts/subscribe-submit.sh"]
+          env:
+            - name: HOST
+              value: {{ include "chronicle.service.name" . }}-chronicle-api
+            - name: PORT
+              value: {{ .Values.port | quote }}
+          volumeMounts:
+            - name: config
+              mountPath: /scripts
+            - name: shared-volume
+              mountPath: /shared
+      containers:
+        - name: jq
+          image: stedolan/jq
```

Comment:
use one container image that we define in this project.

Reply:
Was having issues with dependencies trying to install `jq` in the `node` container. That was part of the idea.